app-backend/common/src/main/scala/cache/CacheClient.scala

```diff
@@ -26,33 +22,28 @@ object CacheClientThreadPool extends RollbarNotifier {
         new ThreadFactoryBuilder().setNameFormat("cache-client-%d").build()
       )
     )
-
-  implicit val system = ActorSystem("rollbar-notifier")
```

Comment:
I don't think these are necessary any more because we are using the Rollbar Java SDK.